persistence/persistence-mybatis/src/main/resources/org/cbioportal/persistence/mybatis/GenePanelMapper.xml

```diff
@@ -291,21 +285,21 @@
         delete from sample_profile  where PANEL_ID = #{panelId} and PANEL_ID is not null
     </delete>
 
-    <insert id="insertGenePanelList" parameterType="map" useGeneratedKeys="false">
+    <insert id="insertGenePanelList" parameterType="map" useGeneratedKeys="true">
```

Comment:
http://www.mybatis.org/mybatis-3/sqlmap-xml.html mentions that with useGeneratedKeys, mybatis expects keyProperty="<fieldname>".  Maybe it can also autodetect if there is a primary key? I guess if this is working all is well.